internal/command/volumes/create.go

```diff
@@ -79,6 +80,43 @@ func runCreate(ctx context.Context) error {
 		return err
 	}
 
+	// If the Yes flag has been supplied we skip the warning entirely, so no
+	// need to query for the set of volumes
+	if !flag.GetYes(ctx) {
+		// fetch the set of volumes for this app. If > 0 we skip the prompt
+		var volumes []api.Volume
+		if volumes, err = client.GetVolumes(ctx, appName); err != nil {
+			return err
+		}
+
+		var matches int32
+		for _, volume := range volumes {
+			if volume.Name == volumeName {
+				matches++
+			}
+		}
+
+		if matches == 0 {
+			var confirmed bool
+			io := iostreams.FromContext(ctx)
+			colorize := io.ColorScheme()
+
+			const msg = "Warning! Individual volumes are pinned to individual hosts. You should create two or more volumes per application. You will have downtime if you only create one."
```

Comment:
I like to include instructions for the developer to solve the issue... maybe something like "make sure to create a second volume with `fly volume create --require-unique-zone` to ensure a second volume exists on a separate worker`. 

alternatively, maybe we make it so they can just create two volumes that will exist on different workers? E.g., `--count 2 ` would create two volumes in the same region and ensure they are on different hosts? that could be done separately / in addition to updating the warning msg.

Reply:
That makes sense - I'll work the additional messaging in.

I'll look into adding the `--count 2` flag in a separate PR if that makes sense?